### PR TITLE
Footer-styling-done

### PR DESCRIPTION
**Pull request by aakankshabhende**
#### Fixes #(issue number)
 Fixes : #35 

#### Describe the changes you've made
I styled the footer in index, about and contact page of the website and linked the items.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA
Screenshot (270)

#### Checklist

- [x] I have mentioned related issue number in PR description.
- [x] I have added screenshots and related demo links of my changes..
- [x] The title of my pull request is a short description of the requested changes.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] Open Source Program names(DCP'21)

### CODE REVIEW

**Comment by QAZIMAAZARSHAD** [comment]
Resolve conflicts. 
Fetch upstream.


Also fix the scroll to top button in the footer.

**Comment by satyampgt4** [comment]
please resolve conflict

**Comment by QAZIMAAZARSHAD** [comment]
Share the website link when you are done so I can review your changes.

**Comment by aakankshabhende** [comment]
Which link @QAZIMAAZARSHAD sir?

**Comment by QAZIMAAZARSHAD** [comment]
Your website link.

I want to review if everything's fine or not.


And if you are very sure that your changes are fine then we can merge your PR right away.

**Comment by aakankshabhende** [comment]
Yes sir I'm sure about it, I have checked its responsiveness too.

**Comment by satyampgt4** [comment]
> Your website link.
> 
> I want to review if everything's fine or not.
> 
> And if you are very sure that your changes are fine then we can merge your PR right away.

try this 
git fetch upstream pull/81/head && git checkout FETCH_HEAD

**Comment by QAZIMAAZARSHAD** [comment]
Looks like you haven't removed the old footer from the about page.

**Comment by aakankshabhende** [comment]
> Looks like you haven't removed the old footer from the about page.

I commented it

**Comment by QAZIMAAZARSHAD** [comment]
> > Looks like you haven't removed the old footer from the about page.
> 
> I commented it

Better to remove it.

File: static/style.css [hunk 1]
```diff
@@ -1,3 +1,10 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+
```
Comment (satyampgt4) [inline static/style.css:7]: closing tag for this body is missing

**Comment by QAZIMAAZARSHAD** [comment]
Also, you have not done changes in the FAQ and offers page footer.

After the required changes do share a link so we can review your changes easily.

**Comment by aakankshabhende** [comment]
Okay sir can I just close this PR and do all the required changes and send you a new PR coz after resolving the conflicts even I'm not able to understand how come closing tag of body is not present as on local system it is working totally fine

**Comment by QAZIMAAZARSHAD** [comment]
Sure

**Pull request closed without merging**